.vscode/tasks.json

```diff
@@ -0,0 +1,16 @@
+// See https://go.microsoft.com/fwlink/?LinkId=733558
+// for the documentation about the tasks.json format
+{
+    "version": "2.0.0",
+    "tasks": [
+        {
+            "type": "shell",
+            "label": "Tests with coverage",
+            "command": "tox -e pytest",
```

Comment:
Should we make this "Tests, lint and docs" and run `tox -p`?

Reply:
yes, done